mslib/msui/mscolab.py

```diff
@@ -628,7 +654,7 @@ def after_authorize(self, emailid, r):
         self.conn = sc.ConnectionManager(self.token, user=self.user, mscolab_server_url=self.mscolab_server_url)
         self.conn.signal_reload.connect(self.reload_window)
         self.conn.signal_new_permission.connect(self.render_new_permission)
-        self.conn.signal_update_permission.connect(self.handle_update_permission)
+        self.conn.signal_update_permission.connect(self.handle_update_permission, self.handle_mscolab_buttons)
```

Comment:
I would call the handle_mscolab_button function manually within the other functions. I am not sure how timing and race conditions work with QT and do not want to, actually :-)